main.py

```diff
@@ -0,0 +1,25 @@
+# function that determines "is a palindrome" and "is not a palindrome" 
+
+def palindrome(string):
+  length = len(string)
+  first = 0
+  last = length -1 
+  status = 1
+  while(first<last):
+    if(string[first]==string[last]):
+      first=first+1
+      last=last-1
+    else:
+      status = 0
+      break
+  return int(status)  
+string = input("Enter the string: ")
+
+print("Method 1")
+status= palindrome(string)
+if(status):
+    print("It is a palindrome ")
+else:
+    print("Sorry! Try again")
+
+palindrome("Hannah")
```

Comment:
Great start on this Nichole! You put an interesting spin on reversing words. Goes to show there is more than one way to do things in Python. Keep exploring! It's time to think about how we can build the code for more complex palindromes. Great start though!